### PR TITLE
Create racadm control tasks and job for dell bios vt control

### PR DESCRIPTION
**Pull request by pengz1**
This feature is to control BIOS VT attribute on Dell servers.
I also merged some base tasks that is unnecessary.
Related taskgraph in:
https://github.com/RackHD/on-taskgraph/pull/125

### CODE REVIEW

File: lib/utils/job-utils/racadm-tool.js [hunk 1]
```diff
@@ -367,5 +347,103 @@ function racadmFactory(
...
+        return this.runCommand(host, user, password, "set iDRAC.IPMILan.Enable 0");
+    };
+
+    /**
+     * Returns a promise with the results or errors of configuring one BIOS attribute
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @return {promise}
```
Comment (yyscamper) [inline lib/utils/job-utils/racadm-tool.js]: Forgot some other arguments.
And it's better to describe the resolved value of promise.

File: lib/utils/job-utils/racadm-tool.js [hunk 2]
```diff
@@ -367,5 +347,103 @@ function racadmFactory(
...
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @param {Object} config - includes user force reboot flag.
+     * @return {promise}
+     */
+    RacadmTool.prototype.disableVt = function(host, user, password, config) {
+        var cmd = "set BIOS.ProcSettings.ProcVirtualization Disabled"; 
+        var rebootFlag = config.forceReboot;
```
Comment (yyscamper) [inline lib/utils/job-utils/racadm-tool.js]: it's not necessary to create a new variable here.

**Comment by yyscamper** [comment]
Still :+1:

**Comment by anhou** [comment]
@pengz1  'vt' is confusion, the naming convention is VT-x, could you help update PR title and PR message. and maybe it could be 'VTx' in function or variable naming in codes.

**Comment by pengz1** [comment]
@RackHD/corecommitters @iceiilin @WangWinson

**Comment by iceiilin** [comment]
:+1:

**Pull request merged**